hack/scripts/aws/verify-organization.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# This script checks that an account-id is under the right organizational root
+# unit to be used in testing.
+
+GREEN="\033[0;32m"
+YELLOW="\033[0;33m"
+CLEAR="\033[0m"
+
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 account-id [aws-profile]"
+    echo "Verify the account with 'account-id' is situated in the right place in the organization structure to be used in testing."
+fi
+
+ACCOUNT_ID="${1}"
+AWS_PROFILE="${2:-osd-staging-1}"
+
+find_parents() {
+    CURRENT_ID=$1
+    CURRENT_TYPE="start"
+    PARENTS=()
+    while [ $CURRENT_TYPE != "ROOT" ]; do
+        echo "Retrieving parents for $CURRENT_ID"
+        parent=$(aws --profile "$AWS_PROFILE" organizations list-parents --child-id "$CURRENT_ID")
```

Comment:
Make sure to either add `--output=json` to the awscli command, or `export AWS_DEFAULT_OUTPUT="json"` to guarantee that aws returns json format.
Otherwise, we're relying on the default output on the system. and speaking from experience, this is not a fun error to debug :)